ZipBuilder/Sources/ShellUtils/ShellUtils.swift

```diff
@@ -42,12 +42,12 @@ extension Shell {
   ///   - outputToConsole: A flag if the command output should be written to the console as well.
   ///   - workingDir: An optional working directory to run the shell command in.
   /// - Returns: A Result containing output information from the command.
-  static func executeCommandFromScript(_ command: String,
-                                       outputToConsole: Bool = true,
-                                       workingDir: URL? = nil) -> Result {
+  public static func executeCommandFromScript(_ command: String,
```

Comment:
No action needed: I'd love to see if we can fix this to only use the proper `Process` command at some point